tests/wicked/advanced/sut/t14_ovs_bridge_xml.pm

```diff
@@ -22,6 +22,7 @@ sub run {
     my ($self, $ctx) = @_;
     my $config = '/etc/wicked/ifconfig/ovs-bridge.xml';
     record_info('Info', 'Create a Bridge interface from Wicked XML files');
+    assert_script_run("systemctl start openvswitch");
     $self->get_from_data('wicked/xml/ovs-bridge.xml', $config);
     assert_script_run('ifdown ' . $ctx->iface());
```

Comment:
No need for this line, or?

Reply:
Related question: Advance-testsuite the static network configuration, before it starts with the actual advanced test stuff like tunnesl/bridges a.s.o?

Reply:
I removed it from before_test and add it in the tests where it's needed (t13, t14) where OVS is tested.

> Related question: Advance-testsuite the static network configuration, before it starts with the actual advanced test stuff like tunnesl/bridges a.s.o?
I am working on it as we have a little mess.

Reply:
Hm, but it doesn't make sense  to `ifdown` a interface when all interfaces are down or?

Reply:
Ah, sure. I thought you were referring to the new line. You are right, and I will remove the ifdown.